src/components/FirmwareUpdate/DownloadingUpdateStep.tsx

```diff
@@ -0,0 +1,32 @@
+import { Flex, Text } from "@ledgerhq/native-ui";
+import React, { useEffect } from "react";
+import { useTranslation } from "react-i18next";
+import FirmwareProgress from "../FirmwareProgress";
+import { track } from "../../analytics";
+
+type Props = {
+  progress?: number;
+};
+const DownloadingUpdateStep = ({ progress }: Props) => {
+  const { t } = useTranslation();
+  
+  useEffect(() => {
```

Comment:
For this cases we have `<Track event="DownloadingFirmware" onMount />` which would do the same, use these when possible

Reply:
Done